From f554e152715a3c06a69954d9d4f15415c798e083 Mon Sep 17 00:00:00 2001
From: Mike Gilbert <floppym@gentoo.org>
Date: Sun, 3 Apr 2016 14:30:02 -0400
Subject: [PATCH] build: Quote systemd variable names

If the systemd-m4 package has been installed, SYSTEMD_USERUNITDIR is
defined as a macro. Quote this name to prevent macro expansion.

Bug: https://bugs.gentoo.org/527432
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 1679a47..4664003 100644
--- a/configure.ac
+++ b/configure.ac
@@ -216,7 +216,7 @@ if (test "${enable_systemd}" != "no" && test -z "${path_systemunitdir}"); then
 	fi
 	AC_MSG_RESULT([${path_systemunitdir}])
 fi
-AC_SUBST(SYSTEMD_SYSTEMUNITDIR, [${path_systemunitdir}])
+AC_SUBST([SYSTEMD_SYSTEMUNITDIR], [${path_systemunitdir}])
 
 AC_ARG_WITH([systemduserunitdir],
 			AC_HELP_STRING([--with-systemduserunitdir=DIR],
@@ -230,7 +230,7 @@ if (test "${enable_systemd}" != "no" && test -z "${path_userunitdir}"); then
 	fi
 	AC_MSG_RESULT([${path_userunitdir}])
 fi
-AC_SUBST(SYSTEMD_USERUNITDIR, [${path_userunitdir}])
+AC_SUBST([SYSTEMD_USERUNITDIR], [${path_userunitdir}])
 
 AC_ARG_ENABLE(datafiles, AC_HELP_STRING([--disable-datafiles],
 			[do not install configuration and data files]),
-- 
2.8.0

